srcpkgs/flowblade/template

```diff
@@ -7,8 +7,7 @@ build_style=python3-module
 make_install_args="--install-lib=${DESTDIR}/usr/share/flowblade"
 hostmakedepends="python3-setuptools"
 # https://github.com/jliljebl/flowblade/blob/master/flowblade-trunk/docs/DEPENDENCIES.md
-depends="frei0r-plugins gmic mlt-python3 python3-Pillow python3-dbus gtk+3
- python3-gobject python3-numpy swh-plugins"
+depends="frei0r-plugins gmic libgirepository librsvg mlt-data mlt-python3 python3-Pillow python3-dbus gtk+3 python3-gobject python3-numpy sox swh-plugins"
```

Comment:
I don't see a reason for this, all of the added dependencies are already dependencies through a package that's already in $depends.

also, commit message is wrong and $revision should be increased

Reply:
>I don't see a reason for this, all of the added dependencies are already dependencies through a package that's already in $depends.

ok but flowblade was not launching and it was saying that module "mlt "was missing, then after installing mlt-python3 it also said that module "gi" was missing, which could be fixed by installing libgirepository

also librsvg mlt-data and sox were also on the dependency page of flowblade so i added them as well while i was at it

>commit message is wrong and $revision should be increased

my bad, im going to fix that quickly

Reply:
> > I don't see a reason for this, all of the added dependencies are already dependencies through a package that's already in $depends.
> 
> ok but flowblade was not launching and it was saying that module "mlt "was missing, then after installing mlt-python3 it also said that module "gi" was missing, which could be fixed by installing libgirepository

`mlt-python3` is already a dependency, I have no idea how you could install `flowblade` without `mlt-python3` getting installed as a dependency. `gi` missing means you were missing python3-gobject which is also already a dependency.


> also librsvg mlt-data and sox were also on the dependency page of flowblade so i added them as well while i was at it

All of these are already dependencies, just not directly.

> > commit message is wrong and $revision should be increased
> 
> my bad, im going to fix that quickly

Please read CONTRIBUTING.md, this PR should contain only one commit because it touches only one package.